perfdash/config.go

```diff
@@ -91,6 +91,20 @@ var (
 		},
 	}
 
+	// benchmarkDescriptions contains metrics exported by test/integration/scheduler_perf
+	benchmarkDescriptions = map[string]TestDescription{
+		"BenchmarkResults": {
+			Name:             "benchmark",
+			OutputFilePrefix: "BenchmarkResults",
```

Comment:
Shouldn't this be BenchmarkResults_benchmark? (To not conflict with the other human-readable file)

Reply:
Nope. I tested, it works fine.
Also, the human-readable file does not have a date suffix and has a .txt extension.

Reply:
Sg.